serde-jackson/src/main/java/io/micronaut/serde/jackson/JacksonConfiguration.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.serde.jackson;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.StreamWriteFeature;
+import com.fasterxml.jackson.core.json.JsonReadFeature;
+import com.fasterxml.jackson.core.json.JsonWriteFeature;
+import io.micronaut.context.annotation.BootstrapContextCompatible;
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.core.annotation.Internal;
+import io.micronaut.serde.config.SerdeConfiguration;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * Configuration for the Jackson.
+ *
+ * @author Denis Stepanov
+ * @since 2.3
+ */
+@BootstrapContextCompatible
+@Internal
+@ConfigurationProperties(JacksonConfiguration.PREFIX)
+public final class JacksonConfiguration {
```

Comment:
rename to `JacksonCoreConfiguration` since we already have a type with this name in core

Reply:
Renamed to `SerdeJacksonConfiguration`